opencti-platform/opencti-graphql/tsconfig.json

```diff
@@ -14,7 +14,7 @@
     "esModuleInterop": true,
     "skipLibCheck": true,
     "forceConsistentCasingInFileNames": true,
-    "importsNotUsedAsValues": "error",
```

Comment:
this option had been deprecated in favor of `verbatimModuleSyntax`. Hence the change. See https://www.typescriptlang.org/tsconfig#importsNotUsedAsValues for more information